### ISSUE TITLE
SCM - better distinguish when repositories have the same name

### ISSUE THREAD

**Issue by RTS007** [microsoft/vscode#126828]
In the company where I work the project structure is as follows: .../PROJ_NR/software/<.git folder>+<data>
In Source Control Repositories only the parent folder is displayed, which results in displaying only 'software' names. Is it possible for SCR to recognize identical folder names and add one more parent folder when it recognizes two identical names? I am adding a picture of how it looks like in practice at the moment. 

![Screenshot_20210621_153656](https://user-images.githubusercontent.com/13761341/122772611-d7fd3880-d2a7-11eb-8c94-435873f61abe.png)

**Comment by lszomoru** [comment]
@RTS007, would it help if there would be an option to display the path of each repository in the repositories view?

**Comment by RTS007** [comment]
> @RTS007, would it help if there would be an option to display the path of each repository in the repositories view?

Yes, any information to distinguish between repositories would be helpful.

**Comment by lszomoru** [comment]
Verification steps:
* Set up the following folder structure
```txt
workspace
 |- project1
 |  |- src <- git repository
 |- project2
 |  |- src <- git repository
```
* Set `git.repositoryScanMaxDepth` to 2
* Open the `workspace` folder in VS Code
* Open the Source Control Repositories view
    * Confirm that descritpion is shown for the two repositories to disambiguate them
